Sources/joyfill-components-swift/Components/DateTime/RPicker.swift

```diff
@@ -0,0 +1,461 @@
+        sv.axis = NSLayoutConstraint.Axis.vertical
+        sv.distribution = UIStackView.Distribution.fill
+        sv.alignment = UIStackView.Alignment.center
+        sv.spacing = 0.0
+        return sv
+    }()
+    
+    private lazy var datePicker: UIDatePicker = {
+        let picker = UIDatePicker()
+        picker.pinConstraints(view, width: view.frame.width)
+        picker.minimumDate = minDate
+        picker.maximumDate = maxDate
+        picker.date = selectedDate
+        picker.datePickerMode = datePickerMode
+        
+        if #available(iOS 14, *) {
+            if datePickerStyle == .Wheel {
+                picker.preferredDatePickerStyle = .wheels
+            } else if datePickerStyle == .Compact {
+                picker.preferredDatePickerStyle = .compact
+            } else {
+                picker.preferredDatePickerStyle = .inline
+            }
+        }
+        return picker
+    }()
+    
+    private lazy var optionPicker: UIPickerView = {
+        
+        let picker = UIPickerView()
+        picker.dataSource = self
+        picker.delegate = self
+        picker.pinConstraints(view, width: view.frame.width)
+        
+        if let selectedIndex = selectedIndex {
+            if (selectedIndex < dataArray.count) {
+                picker.selectRow(selectedIndex, inComponent: 0, animated: false)
+            }
+        }
+        return picker
+    }()
+    
+    private lazy var toolBarView: UIView = {
+        
+        let barView = UIView()
+        barView.pinConstraints(view, height: barViewHeight, width: view.frame.width)
+        
+        //add done button
+        let doneButton = self.doneButton
+        let cancelButton = self.cancelButton
+        
+        barView.addSubview(doneButton)
+        barView.addSubview(cancelButton)
+        
+        cancelButton.pinConstraints(barView, left: 0, top: 0, bottom: 0, width: buttonWidth)
+        doneButton.pinConstraints(barView, right: 0, top: 0, bottom: 0, width: buttonWidth)
+        
+        if let text = titleText {
+            let titleLabel = self.titleLabel
+            titleLabel.text = text
+            barView.addSubview(titleLabel)
+            titleLabel.surroundConstraints(barView, left: buttonWidth, right: -buttonWidth)
+        }
+        
+        doneButton.setTitle(doneText, for: .normal)
+        
+        if let text = cancelText {
+            cancelButton.setTitle(text, for: .normal)
+        } else {
+            cancelButton.isHidden = true
+        }
+        return barView
+    }()
+    
+    private lazy var lineLabel: UILabel = {
+        let label = UILabel()
+        label.backgroundColor = lineColor
+        label.pinConstraints(view, height: lineHeight, width: view.frame.width)
+        return label
+    }()
+    
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.font = UIFont(name: "HelveticaNeue", size: 14)
```

Comment:
So as a default we don't have to use Helvetica we can use the default iOS font SF I believe. If we have already done this more all components then that is fine to keep it

Reply:
Done. I'm using systemFont provided by iOS.